test/events/objectEvent.spec.js

```diff
@@ -39,13 +40,15 @@ describe('unit tests for the ObjectEvent class', () => {
     it('should not use eventTimeZoneOffset from settings if it is not overridden', async () => {
       setup({});
       const o = new ObjectEvent();
-      expect(o.eventTimeZoneOffset).to.be.equal(undefined);
+      expect(o.eventTimeZoneOffset).to.be.equal(
+        getTimeZoneOffsetFromStringOrNumber((new Date()).getTimezoneOffset() / 60),
```

Comment:
This might only pass in the `-1` timezone. So once we leave BST, it could break if run in London.

Reply:
Are you sure?
o.eventTimeZoneOffset is set to the local timezone and `(new Date()).getTimezoneOffset() / 60` retrieve the value (in hours) of the timezoneOffset